src/index.js

```diff
@@ -7,46 +7,54 @@ const mongodb = require("./database/mongo");
 dotenv.config();
 
 // Create a new client instance
-const client = new Client({ intents: [GatewayIntentBits.Guilds] });
+const client = new Client({
+  intents: [
+    GatewayIntentBits.Guilds,
+    GatewayIntentBits.GuildMembers,
+    GatewayIntentBits.GuildMessages,
+    GatewayIntentBits.GuildVoiceStates,
+    GatewayIntentBits.GuildWebhooks,
+  ],
+});
 
 // COMMAND HANDLER
 client.commands = new Collection();
 const foldersPath = path.join(__dirname, "commands");
 const commandFolders = fs.readdirSync(foldersPath);
 
 for (const folder of commandFolders) {
-	const commandsPath = path.join(foldersPath, folder);
```

Comment:
Can you check your prettier.js settings or vscode settings to make sure that these are tabbed or 4 spaces long? Not sure why these are shifting around.

Reply:
![image](https://user-images.githubusercontent.com/27266078/230989525-ac397ad8-2c0f-4721-9673-d488501a9067.png)
It looks properly tabbed to me.